docs/articles_en/openvino-workflow/model-optimization.rst

```diff
@@ -13,57 +10,103 @@ Model Optimization Guide
    model-optimization-guide/weight-compression
 
 
-Model optimization is an optional offline step of improving the final model performance
-and reducing the model size by applying special optimization methods, such as 8-bit
-quantization, pruning, etc. OpenVINO offers three optimization paths implemented in
-`Neural Network Compression Framework (NNCF) <https://github.com/openvinotoolkit/nncf>`__:
+Model optimization means altering the model itself to improve its performance.
```

Comment:
I would keep the statement about the model size.

Reply:
Applied in https://github.com/openvinotoolkit/openvino/pull/25981/commits/853459f89b301b6edb77f8f0c25f2bec6c6371d1